models/demos/t3000/mixtral8x7b/tt/model_config.py

```diff
@@ -181,9 +181,7 @@ def __init__(self, device=None, instruct=False, dummy_weights=False, max_seq_len
             mcast_in0=False,
         )
 
-        self.model_config[
-            "SDPA_DECODE_PROGCFG"
-        ] = ttnn.experimental.operations.primary.transformers.SDPAMultiCoreProgramConfig(
+        self.model_config["SDPA_DECODE_PROGCFG"] = ttnn.SDPAProgramConfig(
```

Comment:
Is this correct? 
Was the multicore program config of SDPA merged into a single program config?

Reply:
Yep, there's just one config type. The current implementation uses a variant between an empty type and a config type. Which is really just a std::optional of `SDPAMultiCoreProgramConfig`. So I changed a variant to an optional and renamed the type to `SDPAProgramConfig` for brevity, because there is no other config type, so there is no need to disambiguate single-core vs multi-core. We can always bring back more config types if and when necessary.